modules/ztp-stopping-the-existing-gitops-ztp-applications.adoc

```diff
@@ -1,12 +1,12 @@
 // Module included in the following assemblies:
 //
-// *scalability_and_performance/ztp-deploying-disconnected.adoc
+// * scalability_and_performance/ztp-far-edge/ztp-upgrading-gitops.adoc
 
 :_content-type: PROCEDURE
 [id="ztp-stopping-the-existing-gitops-ztp-applications_{context}"]
 = Stopping the existing GitOps ZTP applications
 
-Removing the existing applications ensures that any changes to existing content in the Git repository are not rolled out until the new version of the tooling is available.
+Removing the existing applications ensures that any changes to existing content in the Git repository are not rolled out until the new version of the tools are available.
```

Comment:
In my view, "is" is adequate because it relates to the version, which is singular.